tests/editor/editor.js

```diff
@@ -244,6 +272,23 @@ describe( 'Editor', () => {
 				);
 			} );
 		} );
+
+		it( 'should load plugins built in the Editor even if the passed config is empty', () => {
```

Comment:
Missing tests:

1. When both `build.plugins` and `config.plugins` are set.
1. That plugins defined in `build.plugins` can be loaded by names.
1. That it all works when inheriting from `Editor`.